pkgs/development/libraries/speechd/default.nix

```diff
@@ -62,9 +66,15 @@ in stdenv.mkDerivation rec {
     # ++ optional withIvona "--with-ivona"
   ;
 
+  patches = [ ./add_rhvoice_module.patch ./set_nix_defaultModule.patch ];
+
   postPatch = ''
-    substituteInPlace config/speechd.conf --replace "DefaultModule espeak" "DefaultModule ${selectedDefaultModule}"
+    substituteInPlace config/speechd.conf --replace "nixDefault" "${selectedDefaultModule}"
+    substituteInPlace config/speechd.conf --replace "sd_rhvoice" "${rhvoice}/bin/sd_rhvoice"
+    substituteInPlace config/speechd.conf --replace "RHVoice.conf" "${rhvoice}/etc/RHVoice/RHVoice.conf"
+    ${stdenv.lib.optionalString withPico ''
```

Comment:
This conditional is not needed since the svox dependency can be compiled out. RHVoice, on the other hand, cannot so that one should get a conditional. Though, I would move enabling the rhvoice to a NixOS module.

Reply:
I rearranged the optional stuff in patches and postPatch. Is that what you wanted?

Since speech-dispatcher ships with several speech engines, I think including RHVoice in the same way is convenient for the user. Would a NixOS module not make it only available in NixOS?